Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8273,6 +8292,34 @@ def set_feed_last_run(last_run_indicators):
         demisto.setIntegrationContext(last_run_indicators)
 
 
+def set_last_mirror_run(last_mirror_run):
```

Comment:
Add typing.
like 
`# type: (Any, Optional[str], bool) -> Optional[int]`
it is supported in python2

Reply:
Done